homeassistant/components/dlib_face_identify/image_processing.py

```diff
@@ -14,44 +17,121 @@
     PLATFORM_SCHEMA,
     ImageProcessingFaceEntity,
 )
-from homeassistant.core import split_entity_id
+from homeassistant.core import callback, split_entity_id
 import homeassistant.helpers.config_validation as cv
 
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_NAME = "name"
+ATTR_DISTANCE = "distance"
 CONF_FACES = "faces"
 
+
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
-        vol.Required(CONF_FACES): {cv.string: cv.isfile},
+        vol.Required(CONF_FACES): vol.Any({cv.string: cv.isfile}, cv.isdir),
         vol.Optional(CONF_CONFIDENCE, default=0.6): vol.Coerce(float),
```

Comment:
Default in image_processing is 0.8 ([actually 80](https://github.com/home-assistant/core/blob/686e6b8fc383a3c15a1954092a869bc0e65a28c6/homeassistant/components/image_processing/__init__.py#L46)), perhaps we should update here

Reply:
Dlib actually outputs distances between face encodings -- not confidence values. The 0.6 cut-off is the value recommended by developers of dlib and it's in this case more strict than a value of 0.8. The current dlib integration utilizes the "confidence value" as the distance value for dlib so I didn't want to make it a breaking change.

Reply:
OK fair enough, IMO CONF_DISTANCE would have been a better variable name but there you go